osctiny/extensions/packages.py

```diff
@@ -237,13 +237,20 @@ def push_file(self, project, package, filename, data):
         :param filename: Name of file
         :param data: content of file
         :type data: str or open file handle
+        :param comment: Optional comment to use as commit message
+
+        .. versionchanged:: {{ NEXT_RELEASE }}
+
+           Added an optional `comment` argument to be used as the commit message when writing the
+           file.
         """
         path = [self.base_path, project, package, filename]
 
         self.osc.request(
             url=urljoin(self.osc.url, "/".join(path)),
             method="PUT",
-            data=data
+            data=data,
+            params={"comment": comment} if comment else None
```

Comment:
I believe it's sufficient to pass the params this way. If `comment` is `None`, `Osc.handle_params` will filter out the item for us :grin: 
```suggestion
            params={"comment": comment}
```